.taskcat.yml

```diff
@@ -16,13 +16,12 @@ project:
     RemoteAccessCidr: --override--
     S3TAPConfigBucketName: $[taskcat_random-string]-$[taskcat_current_region]
     EKSClusterName: $[taskcat_random-string]
-    NumberOfNodes: 4
-    MaxNumberOfNodes: 4
     TanzuNetUsername: --override--
     TanzuNetPassword: --override--
     TanzuNetApiToken: --override--
     TanzuNetRelocateImages: --override--
     TAPDomainName: --override--
+    TAPClusterArch: --override--
```

Comment:
```suggestion
```

@satya-dillikar GitHub won't let me format the second part of this as a suggestion block, but I think we should update the taskcat config to something like the following so that we're testing both single & multi-cluster architectures for every e2e test? I think we should add a GovCloud test for coverage too per the customer interest that @desbaux previously mentioned. Sound good?

```yaml
tests:
  single-cluster:
    parameters:
      TAPClusterArch: single
    regions:
      - us-east-1
      - us-gov-west-1
  multi-cluster:
    parameters:
      TAPClusterArch: multi
    regions:
      - us-west-2
```

Reply:
@tlindsay42 
yes, I like that. 
But a few developers like me do not have access to us-gov-west-1. That should restrict developers from using taskcat automation.

Probably we can just test one region for each TAPClusterArch. But if you think your CI already has access to GovCloud, we can include it as you said. 
Please let me know and I can push the change.

Reply:
pushed the change. Please run e2e. thanks again.

Reply:
Confirmed- the QS CI framework has a GovCloud account setup for testing (and same for the Greater China Region partition). Lots of projects like EKS and Linux bastion are e2e tested regularly in each to confirm compatibility. Troubleshooting issues would be challenging though since only those with access could reproduce errors, et cetera. Something to think about.

Also, I highly recommend using the optional `--test-names` and `--regions` arguments of `taskcat test run` to limit tests executed for dev/test purposes in an individuals sandbox account. The default behavior is to fire off all async since the original purpose of taskcat was to facilitate CI, which would likely be excessive in most of those cases.

Reply:
Thanks for the info.